certified-connectors/HighGear Workflow/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2050 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "9.1.0",
+    "title": "HighGear Workflow",
+    "description": "HighGear makes it easy to rapidly deploy and securely manage work, at scale. As a highly configurable no-code workflow platform, connecting with HighGear allows you to readily integrate business applications. Get your entire team connected across the enterprise and start building seamless workflows and processes with more visibility, auditability, and real-time measurement of all work activity.",
+    "contact": {
+      "name": "HighGear Support",
+      "url": "https://www.highgear.com/support/",
+      "email": "support@HighGear.com"
+    }
+  },
+  "host": "highgear.example.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/9.1.0/task/ms-schema": {
+      "get": {
+        "tags": [
+          "Task"
+        ],
+        "summary": "Get task field schema",
+        "operationId": "Task_GetTaskSchema",

```

Comment:
@dhands5312 operationIds should be in Camel Case. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please change to "TaskGetTaskSchema". Please fix all operationIds in the swagger file.

Reply:
@Amjed-Ayoub I have corrected all the operationIds to used Camel Case.